lcls2-cc-lib/Library/Devices/LIC/FB_LIC.TcPOU

```diff
@@ -0,0 +1,43 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="FB_LIC" Id="{c48154b5-8669-4368-9217-42352a16b40e}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_LIC
+VAR_IN_OUT
+    stYStage: DUT_MotionStage;
+    fbArbiter: FB_Arbiter;
+    fbFFHWO: FB_HardwareFFOutput;
+END_VAR
+VAR_INPUT
+    stOut: DUT_PositionState;
+    stTarget1: DUT_PositionState;
+    nTransitionAssertionID: UDINT;
+    nUnknownAssertionID: UDINT;
+END_VAR
+VAR
+    fbYStage: FB_MotionStage;
+
+    {attribute 'pytmc' := '
+        pv: MMS:STATE
+        io: i
+    '}
+    fbStates: FB_WFS_States;
```

Comment:
```suggestion
    fbStates: FB_LIC_States;
```

Reply:
Good catch, I knew I was gonna miss one. Resolved in commit 739e16d